packages/grafana-data/src/types/featureToggles.gen.ts

```diff
@@ -257,4 +257,5 @@ export interface FeatureToggles {
   alertingAlertmanagerExtraDedupStageStopPipeline?: boolean;
   newLogsPanel?: boolean;
   grafanaconThemes?: boolean;
+  alertingRuleVersionHistoryRestore?: boolean;
```

Comment:
we use this ff to hide restore functionality (is wip and it will not be finished in this PR)